.github/workflows/pr.lint-lwc.yml

```diff
@@ -0,0 +1,31 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: PR Gate - Lint LWC
```

Comment:
This job lints js files inside lwc folders using the **/lwc/**/*.js glob pattern